Source/Csla/Core/BusinessBase.cs

```diff
@@ -1240,9 +1240,9 @@ public async Task WaitForIdle()
     /// Await this method to ensure business object is not busy.
     /// </summary>
     /// <param name="timeout">Timeout duration</param>
-    public virtual Task WaitForIdle(TimeSpan timeout)
+    public Task WaitForIdle(TimeSpan timeout)
```

Comment:
If this overload doesn't invoke the virtual overload, then the user can't replace this implementation? Only the cancelation token overload?

Can't both public overloads maybe invoke a `protected virtual` that someone can override?

Reply:
Huh? The first Parameter is using the `WaitForIdle(CancellationToken)` method so we have only one method to override.

Reply:
Line 1245 confused me. I got it now.